### PR TITLE
lists added

### PR DESCRIPTION
**Pull request by vaibhav18matere**
simple and stacked list added
preview : https://deploy-preview-12--falconui.netlify.app/components/lists/lists

### CODE REVIEW

File: components/lists/lists.html [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+            <iframe
+                src="https://carbon.now.sh/embed?bg=rgba%280%2C0%2C0%2C1%29&t=3024-night&wt=none&l=javascript&width=853.5&ds=true&dsyoff=20px&dsblur=68px&wc=true&wa=false&pv=56px&ph=56px&ln=false&fl=1&fm=Hack&fs=14px&lh=133%25&si=false&es=2x&wm=false&code=%2520%253Cul%2520class%253D%2522list-group%2520padding-start%2522%253E%250A%2520%2520%2520%250A%2520%2520%2520%2520%2520%253Cli%2520class%253D%2522list-item%2522%253EList%2520Item%253C%252Fli%253E%250A%250A%2520%2520%2520%2520%2520%253Cli%2520class%253D%2522list-item%2522%253EList%2520Item%253C%252Fli%253E%250A%250A%2520%2520%2520%2520%2520%253Cli%2520class%253D%2522list-item%2522%253EList%2520Item%253C%252Fli%253E%250A%250A%2520%2520%2520%2520%2520%253Cli%2520class%253D%2522list-item%2522%253EList%2520Item%253C%252Fli%253E%250A%250A%2520%2520%2520%2520%2520%253Cli%2520class%253D%2522list-item%2522%253EList%2520Item%253C%252Fli%253E%250A%253C%252Ful%253E"
+                style="width:100%; height:300px; border:0; transform: scale(1); overflow:hidden;"
+                sandbox="allow-scripts allow-same-origin">
+            </iframe>
+            </div>
+        <h2>2. Staked List</h2>
```
Comment (manojasarsa) [inline components/lists/lists.html]: A typo here: "Stacked"

**Review by manojasarsa — APPROVED** [review]
Looks good! 👍

**Review by Raghav888 — APPROVED** [review]
Approved

File: components/lists/lists.html (hunk 1 above)
Comment (vaibhav18matere) [reply components/lists/lists.html]: thank you for pointing out

**Pull request merged**